ui/app/AppLayouts/Chat/panels/ChatAnchorButtonsPanel.qml

```diff
@@ -0,0 +1,68 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+
+import utils 1.0
+import shared 1.0
+
+Item {
+    id: root
+
+    property int mentionsCount
+    property int recentMessagesCount
+
+    signal mentionsButtonClicked
+    signal recentMessagesButtonClicked
+
+    implicitWidth: layout.implicitWidth
+    implicitHeight: layout.implicitHeight
+
+    component AnchorButton: StatusButton {
+        Layout.preferredHeight: 38
+        radius: 100
+        spacing: 4
+
+        icon.width: Style.current.primaryTextFontSize + 5
+        icon.height: Style.current.primaryTextFontSize + 5
+
+        topPadding: Style.current.halfPadding
+        bottomPadding: topPadding
+        leftPadding: Style.current.smallPadding
+        rightPadding: leftPadding
+        font.pixelSize: Style.current.primaryTextFontSize
+    }
+
+    RowLayout {
+        id: layout
+
+        anchors.fill: parent
+
+        spacing: Style.current.smallPadding
+
+        AnchorButton {
+            visible: mentionsCount > 0
+            text: mentionsCount > 99 ? "99+" : mentionsCount
```

Comment:
```suggestion
            text: mentionsCount > 99 ? qsTr("99+") : mentionsCount
```

Reply:
I intentionally omitted the `qsTr()` function, since the text represents a number. However, I now realize that it may be necessary for right-to-left languages.

Reply:
Yup, not all languages use arabic digits. That's also the reason why Locale has this: https://doc.qt.io/qt-5/qml-qtqml-locale.html#zeroDigit-prop